packages/core/src/amazonqGumby/chat/controller/controller.ts

```diff
@@ -401,6 +408,20 @@ export class GumbyController {
             result: MetadataResult.Pass,
         })
         this.messenger.sendSkipTestsSelectionMessage(skipTestsSelection, message.tabID)
+        if (!isSelectiveTransformationReady) {
```

Comment:
note: this is just a feature flag, waiting for our backend work to be ready, will merge with this flag until then